### DISCUSSION TITLE
Is there a similar tool for SQL databases only?

### DISCUSSION THREAD

**Discussion by nnnLik** [BeanieODM/beanie#964] [Question]
I actively use Beanie to work with MongoDB and find this library very convenient. I would like to know if there is a similar tool for working with SQL databases that is also minimalistic, has a similar syntax and is easy to customize?

Also, I wonder if it is possible to port Beanie functionality for use in SQL-ORM. How labor intensive will this process be and what will be required?

**Comment by acarreno-houseloan** [comment]
You may want to try [SQLModel](https://github.com/fastapi/sqlmodel)
